components/03-modules/cart-list-item/CartListItem.stories.js

```diff
@@ -7,6 +8,37 @@ import cartListItem from './mocks/cartListItem.json'
 
 storiesOf('Modules/Cart list item', module)
   .add('Default', () => ({
+    components: { App, AlpacaCartListItem },
+    data: () => ({
+      cartListItem,
+      quantity: 1
+    }),
+    template: `
+      <app>
+        <alpaca-cart-list-item
+          :productImage="cartListItem.productImage"
+          :productName="cartListItem.productName"
+          :productPrice="cartListItem.productPrice"
+          :qtyInput="cartListItem.qtyInput"
+          :productDetails="cartListItem.productDetails"
+          :edit="cartListItem.editIcon"
+          :remove="cartListItem.removeIcon"
+          name="name"
+          href="#"
+          :quantity="quantity"
+          @edit="editMethod"
+          @remove="removeMethod"
+          @change="changeMethod"
+        />
+      </app>
+    `,
+    methods: {
+      editMethod: action('Edit'),
+      removeMethod: action('Remove'),
+      changeMethod: action('Change')
```

Comment:
don't seems to work on the deployed demo

Reply:
Update: 
Now should work, because we merged https://github.com/SnowdogApps/alpaca-storybook/pull/82